osu.Game/Screens/Edit/Compose/Components/EditorSelectionHandler.cs

```diff
@@ -48,11 +49,70 @@ private void load()
         /// </summary>
         public readonly Dictionary<string, Bindable<TernaryState>> SelectionSampleStates = new Dictionary<string, Bindable<TernaryState>>();
 
+        /// <summary>
+        /// The state of each sample bank type for all selected hitobjects.
+        /// </summary>
+        public readonly Dictionary<string, Bindable<TernaryState>> SelectionBankStates = new Dictionary<string, Bindable<TernaryState>>();
+
         /// <summary>
         /// Set up ternary state bindables and bind them to selection/hitobject changes (in both directions)
         /// </summary>
         private void createStateBindables()
         {
+            foreach (string bankName in HitSampleInfo.AllBanks)
+            {
+                var bindable = new Bindable<TernaryState>
+                {
+                    Description = bankName.Titleize()
+                };
+
+                bindable.ValueChanged += state =>
```

Comment:
Personal preference: This `ValueChanged` callback is a bit on the longer side. I'd probably split a method.

Reply:
Hmm, I dunno about this. I don't think splitting out is going to do much for readability given what the method is doing..